.travis.yml

```diff
@@ -9,6 +9,6 @@ addons:
       - g++-4.9
 env: CXX=g++-4.9 CC=gcc-4.9
 node_js:
-    - "0.10"
     - "0.12"
```

Comment:
I think I'll keep it here for a bit longer -- the API to make this work was added in v0.12, and it's "possible" there are still some people running it inside companies.